flow/connectors/eventhub/eventhub.go

```diff
@@ -203,10 +199,6 @@ func (c *EventHubConnector) SyncRecords(req *model.SyncRecordsRequest) (*model.S
 		shutdown <- true
 	}()
 
-	eventsPerBatch := int(req.PushBatchSize)
```

Comment:
Should `PushBatchSize` be marked deprecated?

Reply:
yup, will do!

Reply:
but only for eventhub